lib/smack/SmackRep.cpp

```diff
@@ -201,22 +311,22 @@ const Expr* SmackRep::b2p(const llvm::Value* v) {
 const Expr* SmackRep::i2b(const llvm::Value* v) {
   return Expr::fn(I2B, expr(v));
 }
-const Expr* SmackRep::b2i(const llvm::Value* v) {
-  return Expr::fn(B2I, expr(v));
+const Expr* SmackRep::b2i(const llvm::Value* v, unsigned size) {
```

Comment:
`size` need not be a parameter, since it is determined by the type of `v`.

**Note** that `b2i` is just an artifact of SMACK's implementation. The type of `v` is not `bool` — that is not even an LLVM type — it is `iXX`.